cmake/build_common.cmake

```diff
@@ -0,0 +1,192 @@
+if(NOT DEFINED ROOT OR NOT DEFINED ARCH)
+    message(
+        FATAL_ERROR 
+        "Assert: ROOT = ${ROOT}; ARCH = ${ARCH}"
+    )
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/Version.cmake")
+
+set(
+    BUILD
+    1
+)
+
+if(DEFINED ENV{BUILD_NUMBER})
+    set(
+        BUILD
+        $ENV{BUILD_NUMBER}
+    )
+endif()
+
+set(
+    TAG
+    ""
+)
+
+if(DEFINED ENV{TAG})
+    set(
+        TAG
+        "$ENV{TAG}"
+    )
+endif()
+
+set(
+    BUILD_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/../build"
+)
+
+set(
+    PACKAGE_NAME
+    "cryptopp-${VERSION}-${ARCH}-${BUILD}${TAG}"
+)
+
+set(
+    CMAKE_INSTALL_PREFIX
+    ${ROOT}/${PACKAGE_NAME}
+)
+
+if(WIN32)
+    if(NOT ${ARCH} MATCHES x86)
+        set(
+            MSBUILD_ARCH
```

Comment:
Is this a github bug or you forget to remove this variables from build_common?

Reply:
Yup... This is my mistake... I forgot to remove these variables... Thanks! Removed!